hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/source/TestIncrementalInputSplits.java

```diff
@@ -57,6 +64,7 @@ void testFilterInstantsWithRange() {
         .conf(conf)
         .path(new Path(basePath))
         .rowType(TestConfigurations.ROW_TYPE)
+        .skipClustering(true)
```

Comment:
We can config the `skip_xxx` with the option value from the configuration, just like what we do for streaming read.

Reply:
`conf.set(FlinkOptions.READ_STREAMING_SKIP_CLUSTERING, true);` after line 62.

Reply:
done